Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/README.md

```diff
@@ -0,0 +1,1540 @@
+Use the Cofense Intelligence integration to check the reputation of URLs, IP addresses, file hashes, and email addresses.
+This integration was integrated and tested with version 02 of CofenseIntelligenceV2
+
+
+Search for threats associated with an indicator.
+The verdict (Unknown, Benign, Suspicious, Malicious) of each threat is determined by the impact (None, Minor, Moderate, Major) of its associated web locations as detected in cofense ,  along with a threshold value that is being set by the user (when configuring the instance):
+For each Threat: Verdict = if indicator_found: the_impact_of_the_indicator, 
+else max ( list of all the related indicators’ impact)
```

Comment:
maybe we should change this to a quick paragraph for those uninitiated in if-else statments.